scripts/vagrant-linux-priv-cni.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+VERSION="v0.8.2"
+DOWNLOAD=https://github.com/containernetworking/plugins/releases/download/${VERSION}/cni-plugins-linux-amd64-${VERSION}.tgz
+
+function install_cni() {
+	mkdir -p /opt/bin/cni
+	if [[ -e /opt/bin/cni/${VERSION} ]] ; then
+		return
+	fi
+
+	wget -q -O /tmp/cni-plugins.tar.gz ${DOWNLOAD}
+	tar -xf /tmp/cni-plugins.tar.gz -C /opt/bin/cni
```

Comment:
`-z` isn't needed?

Reply:
Not for extracting. https://www.xkcd.com/1168/ 😀

Reply:
Bah, but the paths are all wrong. Should be `/opt/cni/bin`.

Reply:
Fixed.

Reply:
TIL!

Ugh good catch on the paths.